stdlib/public/Platform/tgmath_derivatives.swift.gyb

```diff
@@ -261,3 +262,26 @@ func _${derivative_kind}Erfc(_ x: ${T}) -> (value: ${T}, ${linear_map_kind}: (${
 %    end # if T == 'Float80':
 %  end # for T in ['Float', 'Double', 'Float80']:
 %end # for derivative_kind in ['jvp', 'vjp']:
+
+// Binary functions
+%for T in ['Float', 'Double', 'Float80']:
+%  if T == 'Float80':
+#if !(os(Windows) || os(Android)) && (arch(i386) || arch(x86_64))
+%  end
+@inlinable
+@derivative(of: pow)
+func _vjpPow(_ x: ${T}, _ y: ${T}) -> (value: ${T}, pullback: (${T}) -> (${T}, ${T})) {
+  let value = pow(x, y)
+  return (value, { v in (v * y * pow(x, y - 1), v * value * log(x)) })
```

Comment:
```suggestion
  return (value, { v in (v * y * pow(x, y - 1), v * value * log(max(1, x))) })
```
This makes the implementation consistent with [TensorFlow](https://github.com/tensorflow/tensorflow/blob/v2.2.0-rc0/tensorflow/python/ops/math_grad.py#L1477-L1478) and [autograd](https://github.com/HIPS/autograd/blob/c6f630a5ec18bd30f1485bc0dbbccb8664c77510/autograd/numpy/numpy_vjps.py#L60).

Reply:
thanks for the reference to those implementations 👍. Will update accordingly.

Reply:
I had a closer look and I think that autograd implementation, for instance, does not compute the max but simply avoids to do `log(x)` when x is 0 ( substitutes `x` with `1`).  So it does something like `return (value, { v in (
    v * y * pow(x, y - 1), v * value * log(x.isLessThanOrEqualTo(0) ? ${T}(1) : x)
 ) })`